src/game-data/visitors/winterVisitorReducers.tsx

```diff
@@ -240,13 +240,19 @@ export const winterVisitorReducers: Record<
     harvestExpert: (state, action) => {
         switch (action.type) {
             case "CHOOSE_CARDS":
-                return promptForAction(state, {
+                return promptToHarvest(state);
+            case "CHOOSE_ACTION":
```

Comment:
I'd move this case after `CHOOSE_FIELD` since this action should occur afterwards